src/commands/link.ts

```diff
@@ -1,22 +1,25 @@
 import { Command, log } from "../../deps.ts";
+import { DefaultOptions } from "../commands.ts";
 import { KEY_FILE, writeAPIKey } from "../keyfile.ts";
 import { version } from "../version.ts";
+import { setupLog } from "../log.ts";
 
 /** Link Command.
  * Provided a key, the `link` commands creates
  * a persistent file on the host os to save
  * the API key to. */
-async function linkCommand(options: Options, key: string) {
+async function linkCommand(options: DefaultOptions, key: string) {
+  await setupLog(options.debug);
+
+  log.debug("Key: ", key);
```

Comment:
We have to decide if we prefer the:
```typescript
info(`Logging this variable: ${foo}`)
```
or the:
```typescript
info("Logging this variable:", foo)
```
approach, and use it consistently throughout the CLI.

Reply:
By using 
```ts 
info("Logging this variable:", foo)
``` 
`foo` is displayed with `Deno.inspect` which helps with visibility

Reply:
I'm in favor of the former.

Reply:
Totally

Reply:
@oganexon that is a very good point... Visibility is top priority for a debugger. Thanks for bringing this up.

Reply:
You'll want to use `Deno.inspect` for raw object data to be displayed cleanly in the console. For instance, `Deno.inspect` uses colour codes to indicate the type of the value.